lib/sneakers/publisher.rb

```diff
@@ -11,7 +11,16 @@ def publish(msg, options = {})
       end
       to_queue = options.delete(:to_queue)
       options[:routing_key] ||= to_queue
-      Sneakers.logger.info {"publishing <#{msg}> to [#{options[:routing_key]}]"}
+      begin
+        # exceptions rescued here because msg is not guaranteed to be JSON
+        Sneakers.logger.info {{
+          message: "publishing <#{msg}> to [#{options[:routing_key]}]",
+          options: options,
+          msg: JSON.parse(msg)
```

Comment:
Since this is happening synchronously on every publish, I wonder if it'd be worth exploring adding the Oj gem to handle the JSON parsing. Last I checked, it was significantly more performant than Ruby's native JSON module

Reply:
done!